### ISSUE TITLE
Marker selection fails

### ISSUE THREAD

**Issue by alex-w** [Stellarium/stellarium#3522]
_The bug report was received via email._

### Actual Behaviour
Please find attached a Stellarium Screenshot of the Sky near Abell 520.
![stellarium-002](https://github.com/Stellarium/stellarium/assets/88731/4969dae0-ca29-4944-8f82-b0b2a3bf0a64)

I tried to select the star marked with Marker1 and the selection failed. Instead of the star marked with Marker 1 other stars in the neighbourhood were randomly selected. I could select stars in the neighbourhood, but also sometimes with changing/random results, selecting again other stars in the neighbourhood. I could not find a meaningfull pattern in this odd behaviour.

In cases, were a star had a proper name, selecting the star via the search dialog was always possible.

I use your latest weekly snapshot. All Star catalogues are completely loaded and stars up to mag 18 are shown.

**Comment by alex-w** [comment]
I can confirm problem with selection tool when FOV is very small.

**Comment by alex-w** [comment]
The issue probably related to #3136 and #365

**Comment by gzotti** [comment]
Yes, some stars seem to escape selection. Even if aberration is switched off.

**Comment by gzotti** [comment]
Yes, it's actually a duplicate of #3136.

**Comment by 10110111** [comment]
> near Abell 520

How do I find it? I type "Abell", and the only result I get is "Abell" without a number, and clicking it twice does nothing (why?).

**Comment by alex-w** [comment]
ACO 520 or Abell 520

**Comment by 10110111** [comment]
> ACO 520 or Abell 520

OK, the search for "ACO 520" does find these both, but clicking any of them I get Train Wreck Cluster selected...

**Comment by gzotti** [comment]
Is that wrong? Remember our "Abell" is ACO, not the PN catalog (which should also go into Stellarium!)
And, seeing the result, should we add an icon for galaxy clusters? (And diameter info?)

**Comment by 10110111** [comment]
> Is that wrong?

OK, after googling it appears to be right. But it surely is confusing when you click an object with one name and get an object with another name selected, with no hint that it's the same object. (Yes, enabling _Catalog numbers_ shows "Abell 520", this may be my fault that I had it disabled.)

The case with simply "Abell" search result still looks like a bug though.

**Comment by gzotti** [comment]
Yes, the Abell "hit" is strange. And it seems to open a query that does not end. Adding a number finally lists possible completions.
I then tried "Barnard" to receive many answers. Added any random number 45. This found LDN1744/Barnard 45. The screen label (after selecting the Barnard catalog in the DSO GUI) shows B45, the name in top-left InfoString is "LDN 1744". Probably when searching for a Barnard object and SIMBAD delivers an object that includes the Barnard number in its name list, also B45 should be retrieved as selected object. (Note I searched for "Barnard 45", not "B45" which does not require Simbad to identify. This adds a layer of complexity to resolve...)
